### DISCUSSION TITLE
How can i parse my query in order to get list of tables that used in the query in KQL in c#

### DISCUSSION THREAD

**Discussion by MiriCoh** [yonileibowitz/yonileibowitz.github.io#7] [Q&A] [answered]
How can i parse my query in order to get list of tables that used in the query in KQL in c#?
i got the columns when i wrote it :
code.ResultType.Display.ToString()

**Comment by y0nil** [comment, marked as answer]
technical questions which aren't directly related to the contents of the posts in this repo are more suitable for posting in StackOverflow with the `kusto-query-language` or `kusto` tags.
